Docs/Components-pages/Grids-page.html

```diff
@@ -0,0 +1,93 @@
+            <h2 class="component-demo-heading"> Two Box Grid </h2>
+            <div class="two-box-grid grid">
+                <div class="box-1 flex box"> Box-1 </div>
+                <div class="box-2 flex box"> Box-2 </div>
+                <div class="box-3 flex box"> Box-3 </div>
+                <div class="box-4 flex box"> Box-4 </div>
+            </div>
+            <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522two-box-grid%2520grid%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522box-1%2520flex%2520box%2522%253E%2520Box-1%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522box-2%2520flex%2520box%2522%253E%2520Box-2%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522box-3%2520flex%2520box%2522%253E%2520Box-3%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522box-4%2520flex%2520box%2522%253E%2520Box-4%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+            style="width: 513px; height: 298px; border:0; transform: scale(1); overflow:hidden;"
+            sandbox="allow-scripts allow-same-origin">
+          </iframe>
+        </section>
+        <section class="content-subsection subsection flex" >
+            <h2 class="component-demo-heading"> Three Box Grid</h2>
+           <p></p>
+            <div class="three-box-grid grid">
+                <div class="box-1 flex box"> Box-1 </div>
+                <div class="box-2 flex box"> Box-2 </div>
+                <div class="box-3 flex box"> Box-3 </div>
+                <div class="box-4 flex box"> Box-4 </div>
+                <div class="box-5 flex box"> Box-5 </div>
+                <div class="box-6 flex box"> Box-6 </div>
+            </div>
+            <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522three-box-grid%2520grid%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522box-1%2520flex%2520box%2522%253E%2520Box-1%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522box-2%2520flex%2520box%2522%253E%2520Box-2%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522box-3%2520flex%2520box%2522%253E%2520Box-3%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522box-4%2520flex%2520box%2522%253E%2520Box-4%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522box-5%2520flex%2520box%2522%253E%2520Box-5%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522box-6%2520flex%2520box%2522%253E%2520Box-6%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+  style="width: 513px; height: 335px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
You can avoid using inline-styling as it's not a good practice. Instead, you can make a class for iframe, and give the css for it.